src/engine/GroupBy.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018, University of Freiburg,
```

Comment:
Is this file supposed to be in this PR? If yes it needs a comment (in the header) what it's stubbed form already does? I guess it might handle better error messages for `GROUP BY`?

Reply:
I originally intended to do more parsing of aliases in the SparqlParser, replacing the the mappings from strings to something more abstract, using the AggregateTypes defined in the GroupBy header. I ended up movong that part of the parsing to the CountAvailablePredicates class, as the aliases could theoreticaly  also describe functions (such as (?a + ?b as >c) resulting in uneccessarily complex parsing in the SparqlParser. I removed the files again in the latest commit.